CarAccidentErrorHandling/CarAccidentErrorHandling/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace CarAccidentErrorHandling
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Car car = new Car();
+            try
+            {
+                car.Start();
+                car.Run();
+            }
+            catch (CollisionException collisionException)
+            {
+                Console.WriteLine("Car Accident: " + collisionException.Message);
+                car.Brake();
+            }
+            catch (RoadObstacleException roadObstaclException)
+            {
+                Console.WriteLine("Road Obstacle: " + roadObstaclException.Message);
+                car.Swerve();
+                car.Brake();
+            }
+            catch (PedestrianCollisionException pedestrianException)
+            {
+                Console.WriteLine("Pedestrian Accident: " + pedestrianException.Message);
+                car.Brake();
+            }
+            catch (MechanicalFailureException mechanicalFailureException)
+            {
+                Console.WriteLine("Mechanical Failure: " + mechanicalFailureException.Message);
+            }
+            catch (Exception exception)
+            {
+                Console.WriteLine("An unexpected error occurred: " + exception.Message);
+            }
+            finally
+            {
+                Console.WriteLine("Attempting to bring the car back to a normal state");
+                car.Stop();
+                car.CloseOpenDoorsOrWindows();
```

Comment:
Why are we closing the doors and windows after the accident?

Reply:
Removed it from here.